spec/models/page_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+describe Page do
+  describe "#convert_is_optional_to_boolean" do
+    context "when make_question_optional feature is disabled", feature_make_question_optional: false do
+      it "returns nil" do
+        expect(described_class.new.convert_is_optional_to_boolean).to be nil
+      end
+    end
+
+    context "when make_question_optional feature is disabled", feature_make_question_optional: true do
```

Comment:
`feature_make_question_optional: true` setup the feature flag to be true for this test scenarios